src/wampy.js

```diff
@@ -783,16 +783,13 @@ class Wampy {
      * @private
      */
     _wsOnOpen () {
-        const { helloCustomDetails, authmethods, authid, authextra, serializer, onError, realm } = this._options;
```

Comment:
What's the reason to remove it, as you are still using most of this later? :)

Reply:
In the previous version we were destructuring a lot of properties right at the beginning of the function, even some that were only being used at the bottom and others such as the onError, which benefited from the ._options prefix to explain where they came from when putting it next to the onError from the promise, for example. So I felt this version is easier to read. What do you think? :)

Reply:
okay. I was thinking of something like that. But decided to clarify :)